browser/ui/webui/brave_webui_source.cc

```diff
@@ -38,22 +40,19 @@
 
 namespace {
 
-struct WebUISimpleItem {
-  const char* name;
-  int id;
-};
-
-void AddLocalizedStringsBulk(content::WebUIDataSource* html_source,
-                             const std::vector<WebUISimpleItem>& simple_items) {
-  for (auto simple_item : simple_items) {
-    html_source->AddLocalizedString(simple_item.name, simple_item.id);
+void AddLocalizedStringsBulk(
+    content::WebUIDataSource* html_source,
+    const std::vector<webui::LocalizedString>& localized_strings) {
+  for (auto localized_string : localized_strings) {
+    html_source->AddLocalizedString(std::string(localized_string.name),
```

Comment:
can you please also fix these to call `WebUIDataSource::AddLocalizedStrings` and `WebUIDataSource::AddResourcePaths` directly?

we can now remove `AddLocalizedStringsBulk` and `AddResourcePaths` from this file.